### PR TITLE
Complete website

### PR DESCRIPTION
**Pull request by Yordinia**
# Day 3 submission
## In this pull request we have
- Styled index.html page with sketchy bootstrap and CSS
- Used JS to show and hide contents

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+.a-border {
+  border-style: solid;
+  border-width: 2px;
+  border-radius: 25px 25px 55px 5px/5px 55px 25px 25px;
+}
+
+footer {
+  margin: 1rem;
+}
```
Comment (rloterh) [inline stylesheet.css]: Kindly improve upon the following aspects of your design:
- [x] The footer should always stick to the bottom of the page
- [x] The add button should align to the right within the form
- [x] OPTIONAL
We want to enhance the user experience of this application, and therefore kindly ensure that an **active menu link is highlighted**(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to mimic the provided wireframe template.

**Your design:**            |  **The original design:**
:-------------------------:|:-------------------------:
image |  image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Kamran & Yordi, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- SPA feature properly implemented
- App is running as intended

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

**You can also consider:**
- Live demo link in your ReadMe is broken, kindly endeavor to fix it.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Yordinia** [comment]
Hi Robert, (@rloterh) 
I have seen and implemented the required changes as described in your
comments. And I successfully submitted the pull request again at 8:30 AM
this morning (utc+3). Later my dashboard said that I could successfully
submit my project. Since I only checked my dashboard, I didn't notice there
was no 2nd code reviewer who declared the project was approved and sent me
an email. This is why my partner also faces an issue with only seeing
'request review' in his dashboard. Even submitting the pull request again didn't help.
In short, we did ot receive a second code review.
Me and my partners pull request link for convenience
- https://github.com/Yordinia/Awesome-Book/pull/4
Thank You.

**Comment by rloterh** [comment]
Hi @Yordinia 

I'll get back on the same soon!

Cheers!

**Comment by rloterh** [comment]
@Yordinia 
Can you kindly confirm if your project on the dashboard is currently marked as changes required?

**Comment by rloterh** [comment]
@Yordinia 
It's probably the next reviewer missed a step in the process reviewing your project.

I suggest you report this issue via the Zendesk so that problem regarding unable to change the update your project status on the dashboard is resolved the sooner.

Cheers! 😃
